int32 start_layer
std_msgs/Float32MultiArray input
std_msgs/Float32MultiArray high_resolution_feature
---
std_msgs/Float32MultiArray output
